### DISCUSSION TITLE
How to toggle current_line_blame by default?

### DISCUSSION THREAD

**Discussion by luizkowalski** [NvChad/NvChad#1905] [Q&A] [answered]
hey! first of all, thank you for NvChad! I've wanted to use Neovim for some time but always found it confusing. NvChad takes all that away!

I want to enable `toggle_current_line_blame` by default but I can't get it to work.  I tried to add this to my custom plugins config:

```
require('gitsigns').setup {
  current_line_blame = true,
}
```
but apparently, I can't just do that because when I open Neovim again, I see this error

```
Failed to run `config` for gitsigns.nvim
...cal/share/nvim/lazy/gitsigns.nvim/lua/gitsigns/async.lua:147: attempt to yield across C-call boundary
# stacktrace:
```

Checking the code for NvChad, looks like `require('gitsigns').setup` is already called prior to this so I think doing what I'm trying to is not the right way but I can't find anything in the docs about how to properly set custom options

**Comment by siduck** [comment, marked as answer]
@luizkowalski add this in your custom plugins table

```lua

  {
    "lewis6991/gitsigns.nvim",
    opts = {
      current_line_blame = true,
    },
  },
  ``` 
  
  works here :D

**Reply by luizkowalski** [reply]
worked like a charm, thanks a lot @siduck!
